src/Raven.Server/Documents/Handlers/AI/Agents/AiAgentProcessorForAddOrUpdateAiAgent.cs

```diff
@@ -38,7 +38,15 @@ public override async ValueTask ExecuteAsync()
             cfg.Identifier = EmbeddingsGenerationConfiguration.GenerateIdentifier(cfg.Name);
         
         ValidateConfiguration(context, cfg);
-        
+
+        if (cfg.Persistence?.ConversationIdPrefix != null)
+        {
+            if (cfg.Persistence.ConversationIdPrefix.EndsWith(RequestHandler.IdentityPartsSeparator) == false)
+            {
+                cfg.Persistence.ConversationIdPrefix += RequestHandler.IdentityPartsSeparator;
```

Comment:
this is hardcoding the separator, do we want that? shouldn't the separator be added on execution?

Reply:
done, moved to start conversation